src/main/java/org/repodriller/filter/diff/NoDiffFilter.java

```diff
@@ -1,5 +1,11 @@
 package org.repodriller.filter.diff;
 
+/**
+ * Default filter that accepts diifs on all files.
+ * 
+ * @author ayaankazerouni
+ * @version January 22, 2018
```

Comment:
Do we usually have @version in the class javadoc?

Reply:
No, I guess not.